helm/alfresco-content-services/community_values.yaml

```diff
@@ -71,6 +71,10 @@ repository:
       memory: "3000Mi"
     limits:
       memory: "3000Mi"
+  # -- The startup probe to cover the worse case startup time for slow start containers
+  startupProbe:
+    periodSeconds: 30
+    failureThreshold: 10
```

Comment:
see my comment in conversation.
tl;dr: I think this should not go to the main container